From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Paul Praschl <praschlpaul@g-p.at>
Date: Sun, 2 Aug 2026 18:49:58 +0200
Subject: [PATCH 081/117] dt-bindings: gpio: apple: Add D2333 PMIC GPIO
 controller

The D2333 PMIC used by T8010 iPhone boards exposes 21 GPIOs through
the parent PMIC regmap, while D2365 exposes 24. Add its compatible and
describe both variants under a shared T8010 PMIC GPIO binding.

Signed-off-by: Paul Praschl <praschlpaul@g-p.at>

Orchard-replay-of: ac6c1bf7ed642efba7d31010b12e98b6bfd286ac
---
 .../bindings/gpio/apple,d2365-pmic-gpio.yaml         | 12 +++++++-----
 1 file changed, 7 insertions(+), 5 deletions(-)

diff --git a/Documentation/devicetree/bindings/gpio/apple,d2365-pmic-gpio.yaml b/Documentation/devicetree/bindings/gpio/apple,d2365-pmic-gpio.yaml
index d45829d0f..890509e8b 100644
--- a/Documentation/devicetree/bindings/gpio/apple,d2365-pmic-gpio.yaml
+++ b/Documentation/devicetree/bindings/gpio/apple,d2365-pmic-gpio.yaml
@@ -4,19 +4,21 @@
 $id: http://devicetree.org/schemas/gpio/apple,d2365-pmic-gpio.yaml#
 $schema: http://devicetree.org/meta-schemas/core.yaml#
 
-title: Apple D2365 PMIC GPIO controller
+title: Apple T8010 PMIC GPIO controller
 
 maintainers:
   - Paul Praschl <praschlpaul@g-p.at>
 
 description: |
-  The Dialog D2365 PMIC used in Apple A10 devices provides 24 GPIOs. The
-  controller is accessed through the regmap exported by the parent Apple I2C
-  PMIC device.
+  The Dialog D2333 and D2365 PMICs used in Apple A10 devices provide 21 and
+  24 GPIOs respectively. The controller is accessed through the regmap
+  exported by the parent Apple I2C PMIC device.
 
 properties:
   compatible:
-    const: apple,d2365-pmic-gpio
+    enum:
+      - apple,d2333-pmic-gpio
+      - apple,d2365-pmic-gpio
 
   gpio-controller: true
 
